test_samples/repositories/integration/test-pattern-fallback.ts

```diff
@@ -1 +1 @@
-var x = "gmpdgrdkogmprdmgmrdjmgodrjmmgodirmgmdrmpaosmgmjmgiopdrsjgdioprjgsiopjmgeispjmghprdjkpodersgpdrjmgdrpkgkapkmsfgijksp0ºakgmiropdjgprdkgkdprokpgapjkmfgpes";
\ No newline at end of file
+var gmpdgrdkogmprdmgmrdjmgodrjmmgodirmgmdrmpaosmgmjmgiopdrsjgdioprjgsiopjmgeispjmghprdjkpodersgpdrjmgdrpkgkapkmsfgijksp0ºakgmiropdjgprdkgkdprokpgapjkmfgpes = "";
```

Comment:
The new tool version doesn't count strings for the max line length. 
As consequence, a test that was using the max-line-length pattern started to fail. This change is to fix the test